docs/en-US/component/tree-select.md

```diff
@@ -106,8 +108,8 @@ and please go to the original component to view the documentation.
 | [tree](./tree.md#attributes)            | [tree](./tree.md#method)      | [tree](./tree.md#events)            | [tree](./tree.md#slots)            |
 | [select](./select.md#select-attributes) | [select](./select.md#methods) | [select](./select.md#select-events) | [select](./select.md#select-slots) |
 
-### Own Attributes
+#### Own Attributes
 
-| Name                      | Description                                                                                                         | Type  | Accepted Values | Default |
-| ------------------------- | ------------------------------------------------------------------------------------------------------------------- | ----- | --------------- | ------- |
-| cacheData **(\> 2.2.26)** | The cached data of the lazy node, the structure is the same as the data, used to get the label of the unloaded data | array | —               | —       |
+| Name                | Description                                                                                                         | Type                     | Default |
+| ------------------- | ------------------------------------------------------------------------------------------------------------------- | ------------------------ | ------- |
+| cacheData ^(2.2.26) | The cached data of the lazy node, the structure is the same as the data, used to get the label of the unloaded data | ^[object]`CacheOption[]` | []      |
```

Comment:
`CacheOption` can be type declared.
refer to https://element-plus.org/en-US/component/cascader.html#cascaderprops

Reply:
updated